safe_transaction_service/history/management/commands/setup_service.py

```diff
@@ -114,8 +114,8 @@ def create_task(self) -> Tuple[PeriodicTask, bool]:
         ('0x34CfAC646f301356fAa8B21e94227e3583Fe3F5F', 6399212, '1.1.1'),
     ],
     EthereumNetwork.VOLTA: [
-        # ('0x3E5c63644E683549055b9Be8653de26E0B4CD36E', 0, '1.3.0+L2'),
-        # ('0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552', 0, '1.3.0'),
+        ('0x3E5c63644E683549055b9Be8653de26E0B4CD36E', 11828662, '1.3.0+L2'),
```

Comment:
I think the block number here should be `11942450`

See: https://volta-explorer.energyweb.org/tx/0xb2de05e31b2f1397252d480ee1cbc1ea4f505aa65dce0e8073f548c5e393bb03/internal-transactions